models/hl7-v3/core.ts

```diff
@@ -172,3 +171,16 @@ export class IntervalUnanchored {
     }
   }
 }
+
+function getNumericValueAsString(numericValue: string | number | LosslessNumber): string {
```

Comment:
This is for use with parsed FHIR messages, so probably belongs elsewhere

Reply:
yeah agreed, doesn't really fit into `models` but it was being used by models, I'll remove this from the NumericValue constructor, set it to take in a string and set a string, and see what the impact is

Reply:
removed from constructor, no impact